Frends.FixedWidthFlatFile.ConvertToJSON/Frends.FixedWidthFlatFile.ConvertToJSON/FixedWidthFlatFile.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Frends.FixedWidthFlatFile.ConvertToJSON.Definitions;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+
+#pragma warning disable 1591
+
+namespace Frends.FixedWidthFlatFile.ConvertToJSON
+{
+    public static class FixedWidthFlatFile
+    {
+        /// <summary>
+        /// This is a task that converts given value to json string.
+        /// Documentation: https://github.com/CommunityHiQ/Frends.FixedWidthFlatFile.ConvertToJSON
+        /// </summary>
+        /// <param name="input">What value to convert.</param>
+        /// <returns>{string Data} </returns>
+        public static Result ParseJSON(Input data)
+        {
+            if(data.FileContent == null || data.FileContent.Count <= 0) throw new ArgumentNullException("FileContent not given. Cannot be empty.");
+
+            CultureInfo culture = string.IsNullOrWhiteSpace(data.culture) ? CultureInfo.InvariantCulture : new CultureInfo(data.culture);
+            Lazy<JToken> jToken = new Lazy<JToken>(() => WriteToJToken(data.FileContent, culture));
```

Comment:
I don't think you need lazy initialization here. The JToken should be quite large to cause issues with variable initialization. You can just use normal JToken.